Pendo Feedback/SourceJSONConfig.json

```diff
@@ -0,0 +1,120 @@
+{
+    "Services": [
+        {
+            "Url": "<<ADD_YOUR_PENDO_FEEDBACK_URL>>",
+            "Paging": {
+                "PageSize": 30,
+                "OffsetStart": 0,
+                "OffsetType": "item",
+                "Parameters": {
+                    "Limit": "limit",
+                    "Offset": "start"
+                }
+            },
+            "Endpoints": [
+                {
+                    "Path": "/features",
+                    "Method": "GET",
+                    "QueryParameters": {
+                        "auth-token": "@ApiKey"
+                    },
+                    "ItemType": "Feature",
+                    "Uri": "%[coveo_url]/features/%[id]",
+                    "ClickableUri": "%[app_url]",
+                    "Body": "%[description]",
+                    "Title": "%[title]",
+                    "Date": "%[created_at]",
+                    "IndexingAction": {
+                        "ActionOnItem": "Ignore",
+                        "Condition": "%[is_private] == \"True\""
+                    },
+                    "Metadata": {
+                        "description": "%[description]",
+                        "title": "%[title]",
+                        "filetype": "Feature",
+                        "status": "%[status]",
+                        "resolution": "%[resolution]",
+                        "groups": "%[groups]",
+                        "products": "%[products[*]]",
+                        "is_private": "%[is_private]",
+                        "created_at": "%[created_at]",
+                        "updated_at": "%[updated_at]",
```

Comment:
Should that be used in the ModifiedDate one level higher so that the ModifiedDate field on the indexed items is not defaulting to the current time stamp?

Reply:
Good catch! Fixed.